### PR TITLE
1.10 branding debianfiles

### PR DESCRIPTION
**Pull request by Sporklin**
yay ._.

### CODE REVIEW

File: contrib/debian/dogecoin-qt.install [hunk 1]
```diff
@@ -1,6 +1,6 @@
-usr/local/bin/bitcoin-qt usr/bin
+usr/local/bin/dogecoin-qt usr/bin
 share/pixmaps/bitcoin32.xpm usr/share/pixmaps
 share/pixmaps/bitcoin16.xpm usr/share/pixmaps
 share/pixmaps/bitcoin128.png usr/share/pixmaps
-debian/bitcoin-qt.desktop usr/share/applications
-debian/bitcoin-qt.protocol usr/share/kde4/services/
+debian/dogecoin-qt.desktop usr/share/applications
+debian/dogecoin-qt.protocol usr/share/kde4/services/
```
Comment (patricklodder) [inline contrib/debian/dogecoin-qt.install]: please add a newline here

File: contrib/debian/dogecoind.install [hunk 2]
```diff
@@ -0,0 +1,2 @@
+usr/local/bin/dogecoind usr/bin
+usr/local/bin/dogecoin-cli usr/bin
```
Comment (patricklodder) [inline contrib/debian/dogecoind.install]: please add a newline here

File: contrib/debian/dogecoind.manpages [hunk 3]
```diff
@@ -0,0 +1,3 @@
+debian/manpages/dogecoind.1
+debian/manpages/dogercoin.conf.5
+debian/manpages/dogecoin-cli.1
```
Comment (patricklodder) [inline contrib/debian/dogecoind.manpages]: please add a newline here

File: contrib/debian/examples/dogecoin.conf [hunk 4]
```diff
@@ -119,15 +119,15 @@
...
 
 # User interface options
 
-# Start Bitcoin minimized
+# Start Dogecoin minimized
 #min=1
 
 # Minimize to the system tray
-#minimizetotray=1
+#minimizetotray=1
```
Comment (patricklodder) [inline contrib/debian/examples/dogecoin.conf]: please add a newline here

File: contrib/debian/rules [hunk 5]
```diff
@@ -21,4 +22,4 @@ override_dh_auto_configure:
 	./configure
 
 override_dh_auto_test:
-	make check
+	make check
```
Comment (patricklodder) [inline contrib/debian/rules]: please add a newline here

**Comment by langerhans** [comment]
Probably needs manual merge since git is retarded sometimes and fucks up the line endings. I assume windows/linux issue here.

File: contrib/debian/dogecoin-qt.install (hunk 1 above)
Comment (rnicoll) [inline contrib/debian/dogecoin-qt.install]: A little lost, new line where exactly? At the end of the file? Seems to have one to me :-/

File: contrib/debian/dogecoind.manpages [hunk 6]
```diff
@@ -0,0 +1,3 @@
+debian/manpages/dogecoind.1
+debian/manpages/dogercoin.conf.5
```
Comment (patricklodder) [inline contrib/debian/dogecoind.manpages]: dogercoin => dogecoin

**Comment by patricklodder** [comment]
utACK - all issues solved from what i could find

File: contrib/debian/dogecoind.bash-completion [hunk 7]
```diff
@@ -0,0 +1 @@
+contrib/dogecoind.bash-completion	dogecoind
```
Comment (langerhans) [inline contrib/debian/dogecoind.bash-completion]: That file wasn't renamed.

**Comment by langerhans** [comment]
Thank you :)

**Pull request merged**